automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioSpec.scala

```diff
@@ -85,7 +85,7 @@ class RStudioSpec extends RuntimeFixtureSpec with NotebookTestUtils with RStudio
               rstudioPage.pressKeys("system(\"cat tests.Rmd\")")
               rstudioPage.pressKeys(Keys.ENTER.toString)
 
-              Thread.sleep(25000)
+              Thread.sleep(45000)
```

Comment:
I saw some RStudioSpec failures that seemed like delocalization wasn't happening. I see this in welder conf:
```
sync-cloud-storage-directory-interval = 15 minutes
delocalize-directory-interval = 30 seconds
```
Bumping sleep time from 25s to 45s to hopefully make it more reliable.